ql/src/test/queries/clientpositive/parquet_historical_timestamp.q

```diff
@@ -1,3 +1,4 @@
+--! qt:timezone:Asia/Singapore
```

Comment:
can't we move timezone setup just before your test case?

Reply:
No, it doesn't matter where it is.  It affects the entire q file.

Also, i checked the changed output , the intent of the test remains the same. 
The test was to check the reader output for the timestamp  written in rome time zone .
According to legacy conversion, the timestamp is converted to the reader's timezone. 
As a result ,  the output has moved ahead to 2019-01-01 08:30:30.111111111 which is the correct value.